libs/newsletter-workflow/src/lib/newsletter-workflow.ts

```diff
@@ -1,20 +1,26 @@
 import type { WizardLayout } from '@newsletters-nx/state-machine';
 import { brazeLayout } from './steps/brazeLayout';
 import { cancelLayout } from './steps/cancelLayout';
-import { createNewsletterLayout } from './steps/createNewsletterLayout';
+import { createDraftNewsletterLayout } from './steps/createDraftNewsletterLayout';
 import { descriptionLayout } from './steps/descriptionLayout';
+import { editDraftNewsletterLayout } from './steps/editDraftNewsletterLayout';
 import { finishLayout } from './steps/finishLayout';
 import { identityNameLayout } from './steps/identityNameLayout';
+import { launchNewsletterLayout } from './steps/launchNewsletterLayout';
 import { ophanLayout } from './steps/ophanLayout';
 import { pillarLayout } from './steps/pillarLayout';
 
+// TODO - this list currently contains steps from *all* wizards
+// should separate wizards be separate out?
 export const newslettersWorkflowStepLayout: WizardLayout = {
```

Comment:
I think the answer is yes, there should be a separate `WizardLayout` objects for the "launch" and "create" wizards. As noted above, we'd need the frontend to provide a wizardId so the route knows which objects to find the step in.
One benefit of this is that the stepIds won't need to be completely unique - EG we could have multiple layouts with steps called 'start' and 'finish'.

Reply:
I have separated out the layouts